### DISCUSSION TITLE
Background not displayed on GitHub pages

### DISCUSSION THREAD

**Discussion by Pablobollero** [community/community#65244] [Repositories] [answered]
### Select Topic Area

General

### Body

Hi there...I'm new in this world and I'm learning.

I have the issue that GitHub pages is not deploying my webpage properly. I added a background following github indications but it seem is not displaying my background.

this is my css

#bodyContainer {
    width: 100vw;
    height: 100vh;
    background-image: url("/img/pictures/Banderas-de-Italia-y-Argentina-unidas.jpg");
    background-image: url("../img/pictures/Banderas-de-Italia-y-Argentina-unidas.jpg");
    background-repeat: no-repeat;
    background-size: cover;
}

this below is the link to my css repository

https://github.com/Pablobollero/3erPreEntregaJavaScript/blob/main/css/style.css

any help with it i'll apreciate due i have to give this project to my school where i'm studing.

**Comment by nirav-gajera** [comment, marked as answer]
you need to change the `url()` function to use a relative path. For example, you can change the` url("/img/pictures/Banderas-de-Italia-y-Argentina-unidas.jpg");` to `url("./img/pictures/Banderas-de-Italia-y-Argentina-unidas.jpg");`

**Reply by Pablobollero** [reply]
I solved it..it was because that bloody Double Quotes! hahaha!
